### PR TITLE
MCQ and Result Page added

### PR DESCRIPTION
**Pull request by ashifkhn**
Inspect the deploy log: https://app.netlify.com/sites/xenodochial-kirch-e898c8/deploys/62126526262eed000736b384
Browse the preview: https://deploy-preview-2--xenodochial-kirch-e898c8.netlify.app/

### CODE REVIEW

File: components/mcq/css/cssAnswer.html [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+                  <label for="option1">Color and style sheets</label>
+              </div>
+                            <div class="option1">
+                  <input type="radio" id="option1 "class="radio" name="answer1" checked>
+                  <label for="option1">Cascading style sheets</label>
+              </div>
+                            <div class="option1">
+                  <input type="radio" id="option1 "class="radio" name="answer1">
+                  <label for="option1">None of the above</label>
+              </div>
```
Comment (HarshnaNHaswani) [inline components/mcq/css/cssAnswer.html]: format file

File: components/mcq/html/html.css [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+  padding: 1rem 2rem;
+}
+
+.navitem {
+  text-decoration: none;
+  color: var(--colorWhite);
+  margin: 0.8rem;
+  font-size: 24px;
+}
+
```
Comment (HarshnaNHaswani) [inline components/mcq/html/html.css]: keep repeating code in separate file

File: components/mcq/css/css.css [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+  color: #414141;
+  align-items: center;
+  padding: 1rem 2rem;
+}
+
+.navitem {
+  text-decoration: none;
+  color: var(--colorWhite);
+  margin: 0.8rem;
+  font-size: 24px;
```
Comment (HarshnaNHaswani) [inline components/mcq/css/css.css:25]: use ems instead of px

File: components/mcq/css/css.css [hunk 4]
```diff
@@ -0,0 +1,50 @@
...
+  box-sizing: border-box;
+  display: flex;
+  flex-direction: column;
+  height: 100%;
+  color: var(--colorWhite);
+}
+
+nav {
+  display: flex;
+  /* width: 100%; */
```
Comment (HarshnaNHaswani) [inline components/mcq/css/css.css]: remove comments

File: css/color.css [hunk 5]
```diff
@@ -3,7 +3,7 @@
   --secondaryColor: #2b7a78;
   --lightGrey: #dddddd;
   --colorDarkGreen: #17252a;
-  --colorWhite: #ffffff;
+  --colorWhite: ##ffe6e6;
```
Comment (HarshnaNHaswani) [inline css/color.css:6]: either use rgba or hex

File: css/font.css [hunk 6]
```diff
@@ -1,40 +1,44 @@
...
 }
 
 .textS {
-	font-size: 0.7rem;
+  font-size: 0.7rem;
 }
 
 .fontWhite {
-	color: rgb(255, 236, 236);
+  color: rgb(255, 236, 236);
```
Comment (HarshnaNHaswani) [inline css/font.css:39]: use variable

**Review by HarshnaNHaswani — COMMENTED** [review]
Just a few suggestions:
- change color theme, text is not clear
- highlight right/ wrong option in mcq

File: components/mcq/css/css.html [hunk 7]
```diff
@@ -0,0 +1,83 @@
...
+          <div class="question">
+            <h3 class="qquestion">2.The property in CSS used to change the background color of an element is -</h1>
+          </div>
+          <div class="options">
+              <div class="option1">
+                  <input type="radio" id="option1 "class="radio" name="answer2">
+                  <label for="option1">bgcolor</label>
+              </div>
+                            <div class="option1">
+                  <input type="radio" id="option1 "class="radio" name="answer2">
```
Comment (raieegulhane) [inline components/mcq/css/css.html]: Give proper indentations.

File: components/mcq/css/cssAnswer.html [hunk 8]
```diff
@@ -0,0 +1,81 @@
...
+      <div class="questionContainer p1">
+          <div class="question">
+            <h3 class="qquestion">1. CSS stands for :</h1>
+          </div>
+          <div class="options">
+              <div class="option1">
+                  <input type="radio" id="option1 "class="radio" name="answer1">
+                  <label for="option1"> Cascade style sheets</label>
+              </div>
+                            <div class="option1">
```
Comment (raieegulhane) [inline components/mcq/css/cssAnswer.html]: check indentations here too

File: components/mcq/html/html.css [hunk 9]
```diff
@@ -0,0 +1,50 @@
...
+  height: 100%;
+  color: var(--colorWhite);
+}
+
+nav {
+  display: flex;
+  /* width: 100%; */
+  justify-content: space-between;
+  text-align: center;
+  color: #414141;
```
Comment (raieegulhane) [inline components/mcq/html/html.css:16]: Use variables for colors.

File: components/mcq/react/reactAnswer.html [hunk 10]
```diff
@@ -0,0 +1,81 @@
...
+      </nav>
+
+
+      <div class="questionContainer p1">
+          <div class="question">
+            <h3 class="qquestion">1. Which of the following are the advantages of React.js?</h1>
+          </div>
+          <div class="options">
+              <div class="option1">
+                  <input type="radio" "class="radio" name="answer1">
```
Comment (raieegulhane) [inline components/mcq/react/reactAnswer.html]: double quotes before class on line 37. Class wont work.

File: components/mcq/react/reactAnswer.html [hunk 11]
```diff
@@ -0,0 +1,81 @@
...
+                  <label for="option1">React.js can increase the application's performance with Virtual DOM.</label>
+              </div>
+                            <div class="option1">
+                  <input type="radio" "class="radio" name="answer1">
```
Comment (raieegulhane) [inline components/mcq/react/reactAnswer.html]: same on line 41, 45, 49, 61,  65, 69, 73.

**Review by raieegulhane — COMMENTED** [review]
Check comments. There are some error on line line numbers I have mentions.
In preview, full marks are obtained even when no ans is selected.

File: css/font.css (hunk 6 above)
Comment (ashifkhn) [reply css/font.css:39]: done

File: components/mcq/css/cssAnswer.html (hunk 8 above)
Comment (ashifkhn) [reply components/mcq/css/cssAnswer.html]: fixed

**Pull request merged**